pkgs/development/libraries/jbig2dec/default.nix

```diff
@@ -14,6 +14,11 @@ stdenv.mkDerivation rec {
   '';
 
   preConfigure = ''
+    # Strip ./configure invocation from autogen (which create some required
+    # files), and allow default mechanism to call ./configure with all required
+    # arguments
+    sed -i '$d' autogen.sh
```

Comment:
It seems more appropriate to put this in postPatch. But the patch LGTM otherwise, and locally it seems to work well.

Reply:
Good idea, I'd fix right now